src/sagemaker_xgboost_container/encoder.py

```diff
@@ -18,12 +18,18 @@
 import tempfile
 from typing import Iterable
 
+import mlio
```

Comment:
There seems to be some ambiguity. Would this be correct?
```
from __future__ import absolute_import

import csv
import logging
import os
import tempfile
from typing import Iterable

import mlio
import numpy as np
import xgboost as xgb
from mlio.integ.numpy import as_numpy
from mlio.integ.scipy import to_coo_matrix
from scipy.sparse import vstack as scipy_vstack

from sagemaker_containers import _content_types, _errors
from sagemaker_xgboost_container.constants import xgb_content_types
```

Reply:
The `sagemaker_containers` would go in group 2 (I guess). Also the guideline is to keep it in alphabetical order within a group. Might seem pedantic but leads to consistent import structure in multiple files.

Reply:
So you would recommend mixing the `from` and `import` statements within each group?

Reply:
I'm not sure of the question. Do you mean should all the `import` be together? I would give higher precedence to the alphabetical ordering.

Reply:
```
from __future__ import absolute_import

import csv
import logging
import os
import tempfile
from typing import Iterable

import mlio
from mlio.integ.numpy import as_numpy
from mlio.integ.scipy import to_coo_matrix
import numpy as np
from sagemaker_containers import _content_types, _errors
from scipy.sparse import vstack as scipy_vstack
import xgboost as xgb

from sagemaker_xgboost_container.constants import xgb_content_types
```
I think a case can be made for sagemaker containers being in group 3

Reply:
It looks like in the other files `sagemaker_containers` has generally been placed in group 3 together with `sagemaker_xgboost_container`. How have you been doing it in the MMs integration? Should we settle on a convention now?

Reply:
`sagemaker_containers` in group 3 makes sense.

Reply:
Eric and I agreed that group 2 makes the most sense, since sagemaker_containers is a third party library, not a local module. It just doesn't look as visually pleasing :)